st2api/st2api/controllers/v1/auth.py

```diff
@@ -126,18 +136,42 @@ def get_all(self, requester_user, show_secrets=None, limit=None, offset=0):
 
         return resp
 
+    def checkPersonalAPIKeyPermission(self, api_key_api, requester_user):
+        """
+        Checks whether requested user is the creating/updating/fetching/deleting the api key. This is used when
+        'personal_keys' flag is set of 'rbac' group in conf file.
+        """
+        if not api_key_api or not requester_user:
+            return False
+        rbac_utils = get_rbac_backend().get_utils_class()
+        user_is_admin = rbac_utils.user_is_admin(user_db=requester_user)
+        is_same_user = requester_user.name == api_key_api.user
+        if (user_is_admin) or (is_same_user or api_key_api.user == ""):
+            return True
+        else:
+            return False
```

Comment:
Why apply a conditional test here when the evaluation result can be returned directly?

```python
return (user_is_admin) or (is_same_user or api_key_api.user == "")
```